event_line.m

```diff
@@ -43,23 +43,40 @@ function event_line(temp,MHW,mclim,m90,loc,data_start,date_start,date_end,vararg
 %
 %   'Fontsize' - Default is 16. The size of font in plot.
 
-paramNames = {'Event','Color','Alpha','LineWidth','FontSize'};
-defaults   = {'MHW',[1 0.5 0.5],0.5,2,16};
+    vEvent = 'MHW';
+    vColor = [1 0.5 0.5];
+    vAlpha = 0.5;
+    vLineWidth = 2;
+    vFontSize = 16;
+    
+    paramNames = {'vEvent','vColor','vAlpha','vLineWidth','vFontSize'};
+    defaults   = {'MHW',[1 0.5 0.5],0.5,2,16};
+
+    varargin = reshape(varargin,2,length(varargin)/2);
 
-[vEvent, vColor,vAlpha,vLineWidth,vFontSize]...
-    = internal.stats.parseArgs(paramNames, defaults, varargin{:});
```

Comment:
Once again, this only works if the user has the Stats toolbox. I've removed it similarly to detect, so that users can use it without the toolbox